projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -4528,6 +4528,7 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
 
                 const cell = new IgxCell(id, index, col, rowData[col.field], value, rowData, this as any);
                 this.gridAPI.update_cell(cell);
+                this.validation.markAsTouched(rowSelector, column);
```

Comment:
Not entirely sure on this one, I'm kind of okay with both versions, though the existing behavior seems quite reasonable to me. Might need to check with UX still

Reply:
After discussing it, we decided to not mark them as touched automatically, but expose the API as public so that the user could mark them if he so chooses.